etl/steps/data/garden/health/2023-08-16/deaths_karlinsky.meta.yml

```diff
@@ -0,0 +1,63 @@
+tables:
+  deaths_karlinsky:
+    variables:
+      reg_deaths:
+        title: Registered deaths
+        unit: "deaths"
+        description: >-
+          The number of deaths registered in the vital registration system for the country-year.
+        display:
+          numDecimalPlaces: 0
+
+      expected_wpp:
+        title: Expected number of deaths (WPP 2019)
+        unit: deaths
+        description: >-
+          The number of deaths expected to occur for the country-year from World Population Prospects 2019.
+        display:
+          numDecimalPlaces: 0
+        presentation:
+          title_public: Expected number of deaths
+          title_variant: WPP 2019
```

Comment:
Quick thought: Maybe you should have used `producer_short: WPP` instead of `title_variant`. And I'm not sure if the year is necessary. Anyway, no need to change it for now, just so you consider it while we figure these details out.

Reply:
ha damn ー  I thought I had changed this